File: trunk/PolarMath/src/org/macondo/polar/evaluation/hrv/IN.java
package org.macondo.polar.evaluation.hrv;

import org.macondo.polar.data.Training;
import org.macondo.polar.evaluation.Evaluation;

public class IN implements Evaluation<Integer> {
	
	public Integer evaluate(Training training) {
		double bp = new BP().evaluate(training);
		int amo = new AMoPercents().evaluate(training);
		double mo = new Mo().evaluate(training);
		
		return (int)(amo / (2 * bp * mo));
	}

}

File: trunk/PolarMathCS/PolarMath/Evaluation/HRV/IN.cs
using PolarMath.Data;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace PolarMath.Evaluation.HRV
{
    public class IN : IEvaluation<int>
    {
        public int Evaluate(Training training)
        {
            var bp = training.Evaluate( new BP() );
            var amo = training.Evaluate( new AMoPercents() );
            var mo = training.Evaluate( new Mo() );

            return (int) (amo / (2 * bp * mo));
        }
    }
}
